test/software/amazon/ion/impl/SymbolTableTest.java

```diff
@@ -297,6 +283,50 @@ public void testLocalSymbolTableAppendImportBoundary()
         assertNull(appended.find("o1"));
     }
 
+    @Test
+    public void testMutateDOMAfterLocalSymbolTableAppend() throws IOException {
```

Comment:
Nit for the nit: there is no standard or recommendation for capitalization of acronyms in the Java Language Specification and more examples of `DOM` than `Dom` in the JDK ([`DOMURIReference`](https://docs.oracle.com/javase/6/docs/api/javax/xml/crypto/dom/DOMURIReference.html), [`DOMSource`](https://docs.oracle.com/javase/6/docs/api/javax/xml/transform/dom/DOMSource.html), etc.). The standard within `ion-java` appears to be to camel-case acronyms, but there are counter examples (`IonUTF8`).

If there is a standard for `ion-java`, it might be good to mention it in [`CONTRIBUTING.md`](https://github.com/amzn/ion-java/blob/master/CONTRIBUTING.md)

Reply:
There are definitely a mix of styles in use in ion-java. I'll change to `testMutateDomAfterLocalSymbolTableAppend` here because that style is used within this file. (Head over to `LocalSymbolTableTest` and you'll see the other style in use...)